src/jetson/models/utils/transform.py

```diff
@@ -34,8 +39,15 @@ def base_transform(self, image:np.ndarray):
         Args:
             image - Image to be transformed
         '''
-        x = cv2.resize(image, (self.size, self.size)).astype(np.float32)
+        x = cv2.resize(image, self.size).astype(np.float32)
         if self.mean is not None:
             x -= self.mean
         x = x.astype(np.float32)
         return x
+
+'''
+class InputImageTransform(BaseTransform):
+    def __init__(self, image_size:Tuple[int]):
+        super()
+        self.input_image_size = image_size
+'''
```

Comment:
why is this commented out? if its not  needed then remove, otherwise explain why its commented